instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -230,6 +238,64 @@ def _traced_execute_pipeline(func, instance, args, kwargs):
 
         return response
 
+    def _add_create_attributes(span, args):
+        _set_span_attribute(
+            span, "redis.create_index.index", _value_or_none(args, 1)
+        )
+        # According to: https://github.com/redis/redis-py/blob/master/redis/commands/search/commands.py#L155 schema is last argument for execute command
+        try:
+            schema_index = args.index("SCHEMA")
+        except ValueError:
+            return
+        schema = args[schema_index:]
+        field_attribute = ""
+        # Schema in format:
+        # [first_field_name, first_field_type, first_field_some_attribute1, first_field_some_attribute2, second_field_name, ...]
+        field_types = ["NUMERIC", "TEXT", "GEO", "TAG", "VECTOR"]
```

Comment:
Can we define a constant for this? And also use list comprehension here?
Something like this:

```
field_attribute = "".join(
    f"Field(name: {schema[index - 1]}, type: {schema[index]});"
    for index in range(1, len(schema))
    if schema[index] in _FIELD_TYPES
)
```

Reply:
Done